tests/histograms_test.py

```diff
@@ -65,17 +65,19 @@ def test_compute_frequency_histogram(self, input, expected):
 
     def test_list_to_contribution_histograms(self):
         histogram1 = hist.Histogram(hist.HistogramType.L0_CONTRIBUTIONS, None)
-        histogram2 = hist.Histogram(hist.HistogramType.LINF_CONTRIBUTIONS, None)
-        histogram3 = hist.Histogram(hist.HistogramType.COUNT_PER_PARTITION,
+        histogram2 = hist.Histogram(hist.HistogramType.L1_CONTRIBUTIONS, None)
+        histogram3 = hist.Histogram(hist.HistogramType.LINF_CONTRIBUTIONS, None)
+        histogram4 = hist.Histogram(hist.HistogramType.COUNT_PER_PARTITION,
                                     None)
-        histogram4 = hist.Histogram(
+        histogram5 = hist.Histogram(
             hist.HistogramType.COUNT_PRIVACY_ID_PER_PARTITION, None)
         histograms = hist._list_to_contribution_histograms(
-            [histogram2, histogram1, histogram3, histogram4])
+            [histogram2, histogram1, histogram3, histogram5, histogram4])
```

Comment:
nit: It is a bit odd that the order is different from the numeric suffix of the histograms